images/linux/ubuntu2004.json

```diff
@@ -295,6 +295,18 @@
             ],
             "execute_command": "/bin/sh -c '{{ .Vars }} {{ .Path }}'"
         },
+        {
+            "type": "shell",
+            "script": [
+                "{{template_dir}}/scripts/installers/kotlin.sh"
```

Comment:
no need to create a section for kotlin, you can install it right after java tools